src/main/groovy/com/athaydes/gradle/ceylon/util/CeylonRunner.groovy

```diff
@@ -12,8 +12,8 @@ class CeylonRunner {
 
     static final Logger log = Logging.getLogger( CeylonRunner )
 
-    static void withCeylon( CeylonConfig config, Closure<?> ceylonConsumer ) {
-        String ceylon = CeylonToolLocator.findCeylon( config.ceylonLocation )
+    static void withCeylon( CeylonConfig config, Project project, Closure<?> ceylonConsumer ) {
```

Comment:
In almost all methods, the order of arguments is `project, config` not `config, project`. Small thing, but makes it much nicer to keep it consistent.